integration/types/datatype_test.rb

```diff
@@ -80,15 +80,18 @@ def test_can_insert_all_primitive_datatypes_nil_values
 
       # Create the input
       params = [0]
-      DatatypeUtils.primitive_datatypes.each do |datatype|
+      DatatypeUtils.primitive_datatypes.each do
         params.push(nil)
       end
 
       # Insert into table
       parameters = ["zz"]
-      parameters.push(('a'..'o').to_a)
+      DatatypeUtils.primitive_datatypes.zip('a'..'z') do |datatype, letter|
```

Comment:
Not critical, but since you're not using the datatype arg, you could replace it with '_'. I imagine RubyMine is flagging this with an unused variable warning or some such.